pom.xml

```diff
@@ -141,45 +142,45 @@
         <elasticsearch-spark.version>6.8.3</elasticsearch-spark.version>
         <clickhouse-jdbc.version>0.2</clickhouse-jdbc.version>
         <hbase-spark.version>1.0.0</hbase-spark.version>
+        <mongo-spark.version>2.2.0</mongo-spark.version>
         <commons-lang3.version>3.4</commons-lang3.version>
         <maven-assembly-plugin.version>2.4</maven-assembly-plugin.version>
+        <spark.scope>compile</spark.scope>
     </properties>
```

Comment:
Why is there a change here?

Reply:
Add a spark.scope, which is convenient for switching to "compile" during local debugging, and forgot to switch back to "provided" i will change it back.